### ISSUE TITLE
pyaerocom conda installation

### ISSUE THREAD

**Issue by jgliss** [NordicESMhub/esmconda-recipes#1]
@annefou: thank you for setting up pyaerocom here, so that it can be installed via conda. Since I am not entirely sure yet, how conda recipes work, I created a new branch *jonasg*, where I updated the meta.yaml file, so that it recognises the latest release of pyaerocom (v0.7.0). I also changed the requirements list. I would appreciate if you could please review my changes, and if everything is okay, merge with master (or let me know and I merge)?

Currently, when calling

```conda install -c nordicesmhub -c conda-forge pyaerocom```

it installs the outdated version v0.6.3 (as defined in the current meta.yaml in master). 

Also: would it be possible to automise this process, i.e., that the recipe is automatically updated whenever there is a new release on github?

Cheers, 
Jonas

**Comment by annefou** [comment]
Yes, yes and yes!!!

Feel free to merge. It needs to be fully automated. I had one simple version but when I moved to nordicESMhub, I decided to make it more generic, using what has been done for bioconda.  I hope it will help to share python packages within our community but I did not have time yet...

**Comment by jgliss** [comment]
Thanks a lot! I just merged and tested the installation of pyaerocom v0.7.0 and it works! :+1: :)

One more thing: do we have a way to correct [the currently displayed installation command](https://anaconda.org/NordicESMhub/pyaerocom) from:

``` conda install -c nordicesmhub pyaerocom```

to 


``` conda install -c nordicesmhub -c conda-forge pyaerocom```

?

And please let me know if I should help with the automisation process for the Anaconda cloud (keen to learn more about conda, recipes, channels, etc.).

Cheers

**Comment by jgliss** [comment]
@annefou: I am not sure if I understand how this works at the moment. I updated pyaerocom yesterday (pre-release, v0.7.1) and changed it here accordingly in meta.yaml, but it was not updated [here](https://anaconda.org/NordicESMhub/pyaerocom). I guess, there is another manual step in between? Or did it not work because I published a pre-release?

Thanks for your help!

Jonas

**Comment by annefou** [comment]
No. I removed the automatic generation (it was really messy with things spread within different repositories...). Everything should be done in this esmconda-recipes repo but I did not have time to look at it yesterday.

**Comment by jgliss** [comment]
Okay. Thank you for your fast answer and for updating pyaerocom! Let me know if I can be of any help in automising this.
